### DISCUSSION TITLE
Welcome to 100-Days-of-Code-Mastering-Python Discussions!

### DISCUSSION THREAD

**Discussion by itsmeshibintmz** [itsmeshibintmz/100-Days-of-Code-Mastering-Python#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

This is an open discussion where we all can help each other pursue our dreams.

**Comment by Mrjoe01** [comment]
Where am i starting?

**Comment by itsmeshibintmz** [comment, marked as answer]
@Mrjoe01 
You mean where to start learning?
If that's what you're seeking for, I can suggest a few sources.
1: https://www.youtube.com/watch?v=rfscVS0vtbw&t=940s
2: https://www.udemy.com/course/100-days-of-code/
